starfish/core/spots/DecodeSpots/test/test_composite_codebook.py

```diff
@@ -0,0 +1,238 @@
+from typing import Tuple
+
+import numpy as np
+
+from starfish.core.codebook.codebook import Codebook
+from starfish.core.imagestack.imagestack import ImageStack
+from starfish.core.imagestack.test.factories import create_imagestack_from_codebook
+from starfish.core.spots.DecodeSpots import MetricDistance, PerRoundMaxChannel, SimpleLookupDecoder
+from starfish.core.spots.FindSpots import BlobDetector
+from starfish.core.types import Axes, Features
+
+
+def simple_gaussian_spot_detector() -> BlobDetector:
+    """create a basic gaussian spot detector"""
+    return BlobDetector(
+        min_sigma=1,
+        max_sigma=4,
+        num_sigma=5,
+        threshold=0,
+        measurement_type='max')
+
+
+def composite_codebook() -> Tuple[Codebook, ImageStack]:
+    """
+    Produce an Imagestack representing a composite experiment where the first 2 rounds are
+    multiplexed data and the last round is sequential data.
```

Comment:
do you have a test where we exercise something like what @berl has (multiplexed data and sequential data mixed into one round)?

Reply:
added!